src/views/PermissionSend.vue

```diff
@@ -144,6 +144,13 @@ export default {
         } catch {
           this.symbol = 'ETH'
         }
+      } else if (this.assetChain === 'RBTC') {
```

Comment:
Seems like we are repeating here. 
Why don't we get the token details regardless of what chain? `tokenDetailProviders` allows that

But in either case, I don't think we should be retrieving token details from the chain like this. Otherwise a dapp could trick a user by asking them to approve a fake token that looks like "DAI" or something else

Only if the token address is listed in `cryptoassets` lib should we display the name